allensdk/brain_observatory/behavior/behavior_project_cache.py

```diff
@@ -283,123 +265,45 @@ def get_behavior_session_table(
 
     def get_session_data(self, ophys_experiment_id: int, fixed: bool = False):
         """
-        Note -- This method mocks the behavior of a cache. No files are
-        actually downloaded for local access. Instead, it adds the
-        session id to a csv log. If the "fixed" parameter is true,
-        then the API will first check to ensure that the log is present
-        in the record before pulling the data.
+        Note -- This method mocks the behavior of a cache. Future
+        development will include an NWB reader to read from
+        a true local cache (once nwb files are created).
+        TODO: Using `fixed` will raise a NotImplementedError since there
+        is no real cache.
         """
-        # TODO: Future development will include an NWB reader to read from
-        # a true local cache (once nwb files are created)
-        # For now just check the log if pass `fixed`
-        path = self.get_cache_path(None, self.OPHYS_ANALYSIS_LOG_KEY)
         if fixed:
-            self.logger.warning(
-                "Warning! Passing `fixed=True` does not ensure that the "
-                "underlying data has not changed, as no data are actually "
-                "cached locally. The log will be updated each time the data "
-                "are pulled from the database for tracking purposes.")
-            try:
-                record = pd.read_csv(path)
-            except FileNotFoundError:
-                raise MissingDataError(
-                    "No analysis log found! Add to the log by getting "
-                    "session data with fixed=False.")
-            if ophys_experiment_id not in record["ophys_experiment_id"].values:
-                raise MissingDataError(
-                    f"Data for ophys experiment {ophys_experiment_id} not "
-                    "found!")
-
+            raise NotImplementedError
```

Comment:
an informative message for the transgressor here?

Reply:
Haha I suppose, but I don't know what I'd say other than, "This isn't implemented yet".